test/e2e/contacts.spec.js

```diff
@@ -16,20 +15,19 @@ describe('e2e', () => {
     let dataContract;
 
     let seeds;
-    let dapiClient;
     let faucetPrivateKey;
     let faucetAddress;
 
-    let bobDashClient;
-    let aliceDashClient;
+    let bobClient;
+    let aliceClient;
 
     let bobIdentity;
     let bobContactRequest;
     let aliceIdentity;
     let aliceProfile;
     let aliceContactAcceptance;
 
-    let dataContractDocumentSchemas;
+    let dataContractDocumentDefintions;
```

Comment:
These document definitions are not the definitions set by `contract.setDefinitions()`, correct? What is the right way to refer to these 2 different things - "document definitions" and "definitions"? Seems like confusing terminology.

Reply:
Yeah, agree. Document schemas?

Reply:
Maybe. That's what it was before this change :thinking: